framework/material/Navigator.kt

```diff
@@ -105,6 +105,7 @@ object Navigator {
     enum class Mode(val clicksToOpenItem: Int) {
         BROWSER(2),
         LIST(1),
+        MENU(1),
```

Comment:
1. Browser is the navigator mode for the type browser and the project browser. These exist with the main studio interface, so it makes sense that we don't want to immediately open an item having clicked it once.
2. List is the navigator mode for the subtypes section on the type page. Clicking on one of those subtypes opens the type page for that subtype. When we navigate back to the previous type page, we don't want to have kept the focus on that clicked type because it isn't something we care to have focused.
3. Menu is equivalent to Browser, but it only requires one click to open. We use menu in its own context outside the main studio interface, so having to click twice on the items doesn't make sense. This also follows IntelliJ's behaviour more closely.

Reply:
I feel like the names all describe distinct concepts within a UI, but you may feel differently.

Reply:
We decided to stick with just `BROWSER` and `MENU`, renaming what is currently `LIST` to `MENU`, as it behaves like a dropdown menu / menu bar - a single click takes it out of the view and transitions to a new view.